.github/shared/tsconfig.json

```diff
@@ -2,6 +2,7 @@
 {
   "extends": "@tsconfig/node20/tsconfig.json",
   "compilerOptions": {
+    "target": "ESNext",
```

Comment:
Why is this needed?

Reply:
I recall needing it for some error I was seeing importing from .github/shared but I don't recall specifically. I removed it and it doesn't break build/test/run so I reverted.